jest/setup.js

```diff
@@ -0,0 +1,23 @@
+
+
+import {mockAppEventParams, mockAppEvents} from './mocks';
+
+jest.mock('../src', () => {
+  const nonMocked = jest.requireActual('../src');
+  return {
+    AppEventsLogger: {
+      logEvent: jest.fn(),
+      logPurchase: jest.fn(),
+      setUserID: jest.fn(),
+      AppEventParams: mockAppEventParams,
+      AppEvents: mockAppEvents,
+    },
+    LoginManager: {
+      logInWithPermissions: jest.fn(),
+    },
+    Settings: {
+      initializeSDK: jest.fn(),
+    },
+    LoginButton: nonMocked.LoginButton,
+  };
+});
```

Comment:
could these all be in the mock (and could the mock export the real symbol names) so that the module is fully tested by library users, with just one line?
I thought the react-native-permissions usage (one line in setup) was super clean, for instance:
https://github.com/zoontek/react-native-permissions#testing-with-jest

Reply:
Yes! I'll check it

Reply:
That was an awesome tip, it's more cleaner now!